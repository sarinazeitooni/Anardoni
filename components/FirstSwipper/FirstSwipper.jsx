import React from 'react';
import {Swiper, SwiperSlide} from 'swiper/react';
import Card from "../card/Card";
import Link from 'next/link';
import cardData from "../../data/CardData";
import style from './scss/first-swipper.module.scss';
const FirstSwipper = () => {
    let data = cardData;
    return (
        <div className={style['swipper-container']}>
                <Swiper spaceBetween={1} slidesPerView={3.1}>
                    {data.map((card) =>
                        <SwiperSlide>
                        <Link href={{
                            pathname : '/cards/[id]',
                            query : {id : card.id}
                        }}>
                            <a>
                            <Card id={card.id} title={card.title} subTitle={card.subTitle}
                                  detailImage={card.detailImage} containerImage={card.containerImage}/>
                            </a>
                        </Link>
                        </SwiperSlide>
                    )}
                </Swiper>
        </div>
    )
};
export default FirstSwipper;