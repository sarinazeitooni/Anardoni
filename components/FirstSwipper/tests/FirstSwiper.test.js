import React from 'react';
import { shallow } from 'enzyme';
import FirstSwipper from "../FirstSwipper";
import {Swiper} from "swiper";

describe("Card component", () => {
    it("should have one Card component", () => {
        const firstswiper = shallow(<FirstSwipper/>);
        expect(firstswiper).toMatchSnapshot();
    });
});