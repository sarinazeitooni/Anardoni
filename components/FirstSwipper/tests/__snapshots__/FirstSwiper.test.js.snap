// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should have one Card component 1`] = `ShallowWrapper {}`;
